music_assistant/controllers/streams.py

```diff
@@ -56,6 +57,18 @@ def __init__(self, mass: MusicAssistant):
     @property
     def base_url(self) -> str:
         """Return the base url for the stream engine."""
+
+        """
+        This is a purpously undocumented feature to override the automatic
+        generated base_url used by the streaming-devices.
+
+        If you need this, you know it, but you should probably try to not set it!
+        Also see https://github.com/music-assistant/hass-music-assistant/issues/802
+        and https://github.com/music-assistant/hass-music-assistant/discussions/794#discussioncomment-3331209
+        """
+        if BASE_URL_OVERRIDE_ENVNAME in os.environ:
+            return os.environ[BASE_URL_OVERRIDE_ENVNAME]
```

Comment:
insert in-line comment here (so appended with #)

Reply:
You mean changing the whole """ style comment to # style?

Reply:
correct and put it between line 69 and 70 to make it clear it belongs to that line